generators/client/templates/angular/src/main/webapp/app/shared/directive/_sort-by.directive.ts

```diff
@@ -15,7 +14,27 @@ export class <%=jhiPrefixCapitalized%>SortByDirective {
         this.<%=jhiPrefix%>Sort = <%=jhiPrefix%>Sort;
     }
 
-    onClick() {
-        this.jhSort.sort(this.jhSortBy);
+    @HostListener('click') onClick() {
```

Comment:
@VictorADS @sendilkumarn I resolved the conflicts here but I'm not sure which was correct as both you guys have mad entireley different changes here and in the sort-directive